<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementDialog</class>
 <widget class="QDialog" name="MeasurementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" rowspan="4">
    <widget class="QGroupBox" name="groupBoxPSI5Test">
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>380</height>
      </size>
     </property>
     <property name="title">
      <string>PSI5下测试</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutPSI5Test">
      <item>
       <widget class="QCheckBox" name="checkBoxReadPhaseInfo">
        <property name="text">
         <string>读取产品phase信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxReadZeroNoise">
        <property name="text">
         <string>读取产品传感器输出零位噪声</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutZeroNoise">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSamplePointsZeroNoise">
          <property name="text">
           <string>采样点数</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditSamplePointsZeroNoise"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelProductType">
          <property name="text">
           <string>产品类型</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditProductType"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxReadManchesterCodeCurrent">
        <property name="text">
         <string>读取产品曼码电流</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutManchesterCodeCurrent">
        <item row="0" column="0">
         <widget class="QLabel" name="labelBaseCurrent">
          <property name="text">
           <string>Base</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditBaseCurrentMin"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelBaseCurrentUnit">
          <property name="text">
           <string>mA -</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEditBaseCurrentMax"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelDeltaCurrent">
          <property name="text">
           <string>Delta</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditDeltaCurrentMin"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelDeltaCurrentUnit">
          <property name="text">
           <string>mA -</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEditDeltaCurrentMax"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxMeasureBitTime">
        <property name="text">
         <string>测量bit time</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutBitTime">
        <item>
         <widget class="QLabel" name="labelBitTime">
          <property name="text">
           <string>Bit Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditBitTime"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxMeasureTDelay">
        <property name="text">
         <string>测量t delay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxSyncPulseVDD">
        <property name="text">
         <string>同步脉冲VDD Base可调</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutSyncPulseVDD">
        <item>
         <widget class="QLabel" name="labelSyncPulseVDD">
          <property name="text">
           <string>VDD Base</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSyncPulseVDD"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAsynchronousMode">
        <property name="text">
         <string>异步通信模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDualTimeSlotMode">
        <property name="text">
         <string>双时间槽通信模式</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxWriteOTP">
     <property name="title">
      <string>烧写OTP</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutWriteOTP">
      <item>
       <widget class="QCheckBox" name="checkBoxWriteOTP">
        <property name="text">
         <string>烧写OTP</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutWriteOTP">
        <item>
         <widget class="QLabel" name="labelIDATA">
          <property name="text">
           <string>IDATA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIDATA"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBoxFunctionalSafetyCalibration">
     <property name="title">
      <string>功能安全标定</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutFunctionalSafetyCalibration">
      <item>
       <widget class="QCheckBox" name="checkBoxFunctionalSafetyCalibration">
        <property name="text">
         <string>功能安全标定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditFunctionalSafety"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBoxScaleCalibration">
     <property name="title">
      <string>标度标定</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutScaleCalibration">
      <item>
       <widget class="QCheckBox" name="checkBoxScaleCalibration">
        <property name="text">
         <string>标度标定</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutScaleCalibration">
        <item row="0" column="0">
         <widget class="QLabel" name="labelTimeInterval">
          <property name="text">
           <string>时间间隔</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditTimeInterval"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelSamplePoints">
          <property name="text">
           <string>采样点数</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditSamplePoints"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelAxis">
          <property name="text">
           <string>采集轴向</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditAxis"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxFrequencyScreening">
     <property name="title">
      <string>频率筛选</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutFrequencyScreening">
      <item>
       <widget class="QCheckBox" name="checkBoxFrequencyScreening">
        <property name="text">
         <string>频率筛选</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutFrequencyScreening">
        <item>
         <widget class="QLabel" name="labelOscillatorFrequency">
          <property name="text">
           <string>OSC输出频率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOscillatorFrequency"/>
        </item>
        <item>
         <widget class="QLabel" name="labelOscillatorFrequencyUnit">
          <property name="text">
           <string>MHz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBoxElectricalTest">
     <property name="title">
      <string>电学测试</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutElectricalTest">
      <item>
       <widget class="QCheckBox" name="checkBoxPinContinuity">
        <property name="text">
         <string>管脚通断测试</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutPinContinuity">
        <item row="0" column="0">
         <widget class="QLabel" name="labelVoltageDrop">
          <property name="text">
           <string>压降上限</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditVoltageDropMax"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelVoltageDropMin">
          <property name="text">
           <string>压降下限</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEditVoltageDropMin"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPowerOnCurrent">
        <property name="text">
         <string>上电电流测试</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutPowerOnCurrent">
        <item row="0" column="0">
         <widget class="QLabel" name="labelVDDIVoltage">
          <property name="text">
           <string>VDDI电压</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditVDDIVoltage"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelVDDICurrent">
          <property name="text">
           <string>电流</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditVDDICurrent"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
